### DISCUSSION TITLE
MD3 Switches in GitHub Mobile app

### DISCUSSION THREAD

**Discussion by Wojtaz0w** [community/community#61958] [Mobile]
Could you make switches use Material Design 3?

**Comment by queenofcorgis** [comment]
Hi @Wojtaz0w ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on MD3. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by Wojtaz0w** [reply]
Sir I meant switches in Github mobile app 😅Should have been clearer 🙂

**Comment by Wojtaz0w** [comment]
Sir i meant switches in Github mobile app 😅

**Reply by queenofcorgis** [reply]
Hi there - can you please update your title and comment to reflect that?

**Reply by Wojtaz0w** [reply]
Done 🙂

**Reply by queenofcorgis** [reply]
Thank you very much!
